editor/components/block-inspector/index.js

```diff
@@ -24,8 +25,21 @@ const BlockInspector = ( { selectedBlock, count } ) => {
 		return <span className="editor-block-inspector__no-blocks">{ __( 'No block selected.' ) }</span>;
 	}
 
+	const blockType = getBlockType( selectedBlock.name );
+
 	return (
-		<Slot name="Inspector.Controls" />
+		<div>
```

Comment:
Should we use a fragment or array instead of an empty `<div>`?

Reply:
I don't know, I tend to prefer adding a wrapper for the root of the components when it doesn't affect styling or such. But no strong argument really.